packages/boost-histogram/meta.yaml

```
@@ -0,0 +1,20 @@
+package:
+  name: boost-histogram
+  version: 1.3.0
+source:
+  url: https://files.pythonhosted.org/packages/0a/9a/1ebde600886eee433a99434e0ddb462630a0b25ab1f58efb91ffc385a0ff/boost_histogram-1.3.0.tar.gz
+  sha256: 62240dae889cbf05f8b4f2203f0b7d6c7a2e029502723a377fc33226e425ce50
+requirements:
+  run:
+    - numpy
+build:
+  script: |
+    pip install setuptools_scm[toml]
+test:
+  imports:
+    - boost-histogram
```

```suggestion
    - boost_histogram
```

Will test locally tomorrow.

---

Well `boost-histogram` definitely isn't right here because `import boost-histogram` is a syntax error.

---

Is there a way to run the tests locally? `make` seems to just build.

---

Maybe:
```sh
PYODIDE_PACKAGES="boost-histogram" make -C packages
pytest -k "firefox and histogram" packages/test_packages_common.py
```